packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@web-chapter/ui",
-    "version": "0.0.42",
+    "version": "0.0.43",
```

Comment:
you might need to merge with main before merging this PR, the current version is 43